File: backend/member/src/main/java/com/olock/blockstock/member/domain/award/dto/ContestEndMessage.java
package com.olock.blockstock.member.domain.award.dto;

import lombok.Getter;

import java.util.List;

@Getter
public class ContestEndMessage {
    private String message;
    private Long contestId;
    private String contestTitle;
    private List<Long> memberIds;
    private List<Long> results;
}

File: backend/tactic-background/domain/contest/schemas/contest_end_message.py
from typing import Optional


class ContestEndMessage:
    message: Optional[str] = None
    contestId: Optional[int] = None
    contestTitle: Optional[str] = None
    memberIds: Optional[list] = None
    results: Optional[list] = None

    def to_dict(self):
        return {
            "message": self.message,
            "contestId": self.contestId,
            "contestTitle": self.contestTitle,
            "memberIds": self.memberIds,
            "results": self.results
        }
